### DISCUSSION TITLE
Off-topic: Connecting with people archiving old photos of places?

### DISCUSSION THREAD

**Discussion by iangilman** [openseadragon/openseadragon#2562] [General]
This isn't about OpenSeadragon, but I figure there are a lot of people in the archival community here, so I thought I'd ask.

I have a large cache of my grandparents' vacation photos from the mid 1900s. Many of them are just of places, not necessarily people. Some of the locations of the photos are: Europe (Switzerland, Germany), Hawaii (Oahu, Maui), Canada (Straight of Georgia), Caribbean (Martinique), etc.

I wonder if there are people out there who collect such as part of a record of how places have changed over time. There are only so many photographs from back then, and you can't make new ones! 

Any ideas on how I might connect with such people? I've been asking around and people have suggested contacting the local historical societies (or museums or colleges) of the various places featured. I'm wondering is there a network or registry of historical societies or if I should just Google "<place name> historical society"?

@SamHall unfortunately none of the photos are from Chattanooga, otherwise I definitely would hit you up with them! But maybe you know where photo archivists from other places hang out?

Thank you!

**Comment by iangilman** [comment]
To the kind person who posted a response here and then deleted it, I'm not sure why you deleted it, but thank you for the suggestions!

**Reply by Benomrans** [reply]
Thank you so much for your kind response. Well, I realized that I misunderstood the issue and my suggestion was irrelevant, since the pictures are yet to be scanned and categorized.

**Reply by Benomrans** [reply]
Have asked around and I think donating the pictures to the American Geographical Society Library would be a good option, as I learned that they would process the assets themselves. ( The delivery would be another issue to figure out ).
their website :  https://uwm.edu/libraries/agsl/
email : agsl@uwm.edu

**Reply by iangilman** [reply]
Wonderful, thank you for the suggestion! I have now contacted them… We'll see what they say!

**Comment by SamHall** [comment]
Thank you for thinking of me. I had missed this post previously.
Specific areas likely have established historical societies, but such a broad swath of tourist areas aren't likely represented by one group. There are 'now and then' and vintage photo groups on Reddit, Facebook, and others. I'm sure you'd like to see something done with them other than being tossed into a drawer. If I see any prolific user of such - will let you know. Otherwise, you could sell them on eBay and hope they land in a good home.

**Reply by iangilman** [reply]
Cool, thank you for the suggestions 😊

**Comment by AdamChomsy** [comment]
Just wondering if there are any updates and whether the collection has been published yet.

**Reply by iangilman** [reply]
Not really... I've been going through them and sorting out the ones I want to keep (since they have my family member in them), but I haven't figured out what to do with the rest of them (that are more just landscapes and other people). So far none of the leads I've chased have pulled up anything.

Thanks for checking in... do you have any ideas?

**Reply by AdamChomsy** [reply]
Unfortunately I don't. I would happily scan the collection myself if living in your neighborhood, but I'm at the other side of the world.
